dbms/tests/integration/test_reload_max_table_size_to_drop/test.py

```diff
@@ -0,0 +1,49 @@
+import time
+import pytest
+import os
+
+from helpers.cluster import ClickHouseCluster
+
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance('node', config_dir="configs")
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+CONFIG_PATH = os.path.join(SCRIPT_DIR, './_instances/node/configs/config.xml')
+
+
+@pytest.fixture(scope="module")
+def start_cluster():
+    try:
+        cluster.start()
+        node.query("CREATE TABLE test(date Date, id UInt32) ENGINE = MergeTree() PARTITION BY date ORDER BY id")
+        yield cluster
+    finally:
+        cluster.shutdown()
+
+
+def test_reload_max_table_size_to_drop(start_cluster):
+    node.query("INSERT INTO test VALUES (now(), 0)")
+
+    time.sleep(5)  # wait for data part commit
```

Comment:
It's commited in INSERT query, sleep is not needed

Reply:
Without some time between INSERT and WAIT table will be successfully dropped with any value in max_table_size_to_drop. Due to the table size calculation algorithm(only committed parts summed), I assumed that it become committed in some time after INSERT.